jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -455,6 +455,10 @@ private void extractQueryParameters()
     /* ------------------------------------------------------------ */
     private void extractContentParameters()
     {
+        // Content cannot be encoded
+        if (_metaData!=null && getHttpFields().contains(HttpHeader.CONTENT_ENCODING))
```

Comment:
this means that all requests with the header "Content-Encoding" will be aborted, right? Not only with a specific value....